src/Sylius/Bundle/ResourceBundle/Controller/SingleResourceProvider.php

```diff
@@ -26,14 +26,31 @@ public function get(RequestConfiguration $requestConfiguration, RepositoryInterf
         if (null !== $repositoryMethod = $requestConfiguration->getRepositoryMethod()) {
             $callable = [$repository, $repositoryMethod];
 
+            if (!method_exists($repository, $repositoryMethod)) {
```

Comment:
If we are using an `InvalidArgumentException` anyway, won't it be cleaner to use [webmozart/assert](https://github.com/webmozart/assert)?